### PR TITLE
Added snackbar

### PR DESCRIPTION
**Pull request by nishasen**
image

### CODE REVIEW

**Comment by nishasen** [comment]
https://deploy-preview-20--nishasen-nembus-ui.netlify.app/html_pages/snackbar

File: Stylesheet/Components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* This is scackbar css file */
+
+.snack-container, .appear, #basic-snackbar.show {
```
Comment (shashidwivedi) [inline Stylesheet/Components/snackbar.css:3]: Is this usage of id intentional, or you will refactor it later?

**Review by shashidwivedi — APPROVED** [review]
rest looks great, good to merge

File: Stylesheet/Components/snackbar.css (hunk 1 above)
Comment (nishasen) [reply Stylesheet/Components/snackbar.css:3]: I have added some js code to show the snackbar on button click , that's why given the id

**Review by vaibhav-mahalle — APPROVED** [review]
Best practices followed during writing the code. Everything looks good to me.

**Pull request merged**